cli/app/app.go

```diff
@@ -854,8 +854,25 @@ func loadComposeFile(file string, opt convertOptions) KomposeObject {
 
 	// transform composeObject into komposeObject
 	composeServiceNames := composeObject.ServiceConfigs.Keys()
+
+	// volume config and network config has not supported
+	if len(composeObject.NetworkConfigs) > 0 {
+		logrus.Warningf("Unsupported network configuration of compose v2 - ignoring")
+	}
+	if len(composeObject.VolumeConfigs) > 0 {
+		logrus.Warningf("Unsupported volume configuration of compose v2 - ignoring")
+	}
+
+	count := 0
 	for _, name := range composeServiceNames {
 		if composeServiceConfig, ok := composeObject.ServiceConfigs.Get(name); ok {
+			//FIXME: networks always contains one default element, even it isn't declared in compose v2.
+			if len(composeServiceConfig.Networks.Networks) > 0 &&
+				strings.Compare(composeServiceConfig.Networks.Networks[0].Name, "default") != 0 &&
```

Comment:
With the sample of default network as below, I see the message still makes sense:

```console
$ cat default-network.yml
version: '2'

services:
  web:
    build: .
    ports:
      - "8000:8000"
  db:
    image: postgres

networks:
  default:
    # Use a custom driver
    driver: custom-driver-1
$ kompose convert --file default-network.yml
WARN[0000] Unsupported network configuration of compose v2 - ignoring 
WARN[0000] Unsupported key build - ignoring             
WARN[0000] [db] Service cannot be created because of missing port. 
file "web-svc.json" created
file "db-deployment.json" created
file "web-deployment.json" created
```
I appreciate your idea of detailing messages. But with default or custom network, from my viewpoint, I see the message `unsupported key networks` is still good. So if voting, I am 0.